streethawk.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
   s.name                = "streethawk"
   s.header_dir          = "StreetHawkCore"
-  s.version             = "1.10.1-beta1"
+  s.version             = "1.10.1-beta+20180413105201"
```

Comment:
It's fine to submit this to our own repo. But please pay attention, not run 
pod trunk push streethawk.podspec which submit to pod repo.
If we run pod push accidentally,  the pod spec will go public and the customer can see a "beta" version. 
When customer run "pod update", current version in pod repo will be used regardless of "beta".

Reply:
No, it won't, did you try by yourself? please check this https://github.com/CocoaPods/CocoaPods/issues/7086 and I did test it on my own pod, with a suffix cocoapod will treat it as a pre-release version which you have to explicitly gives the version number to use that version of pod.

Reply:
OK, just test again. It's same as you described. So Pod fixed this issue. It didn't behave this when I created pod two years ago. Thanks, good update.